package.json

```diff
@@ -31,6 +31,7 @@
   },
   "devDependencies": {
     "@peculiar/webcrypto": "^1.1.7",
+    "@rollup/plugin-node-resolve": "^13.0.0",
```

Comment:
I think you also need commonjs plugin. And btw what dependency do u want to use?